qemu/tests/cfg/blockdev_commit_reboot.cfg

```diff
@@ -0,0 +1,34 @@
+- blockdev_commit_reboot:
+    type = blockdev_commit_reboot
+    virt_test_type = qemu
+    only Linux
+    start_vm = yes
+    kill_vm = yes
+    storage_pools = default
+    storage_type_default = "directory"
+    storage_pool = default
+    image_size_data = 500M
+    image_name_data = data
+    snapshot_tags = sn1 sn2 sn3 sn4
+
+    image_size_sn1 = 20G
```

Comment:
we can use system image's size for all snapshots, so no need to define snx's size here

Reply:
Have updated, thanks for pointing it out.